src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -70,11 +77,15 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
    * Checks if NativeAdapter can play a given mime type.
    * @function canPlayType
    * @param {string} mimeType - The mime type to check
+   * @param {boolean} preferNative - prefer native flag
```

Comment:
you can annotate the default value true
`@param {boolean} [preferNative=true] - prefer native flag
`

Reply:
Nice, didn't know this 👍